packages/code-du-travail-frontend/src/search/search.service.js

```diff
@@ -6,20 +6,19 @@ const {
   publicRuntimeConfig: { API_URL, SUGGEST_URL }
 } = getConfig();
 
-const fetchResults = endpoint => async (query = "", excludeSources = "") => {
-  const url = `${API_URL}/${endpoint}?q=${encodeURIComponent(
+const fetchSearchResults = async ({ query = "", excludeSources = "" }) => {
+  const url = `${API_URL}/search?q=${encodeURIComponent(
     query
   )}&excludeSources=${encodeURIComponent(excludeSources)}`;
   const response = await fetch(url);
   if (!response.ok) {
-    throw new Error("Un problème est survenu.");
+    return { statusCode: response.status };
```

Comment:
I think we should keep the `throw`. its much more explicit and force us to handle it properly

Reply:
sure